constants.mojo

```diff
@@ -0,0 +1 @@
+alias pi = 3.141592653589793
```

Comment:
This is [already supplied](https://docs.modular.com/mojo/stdlib/math/constants/) by the `math` package in the stdlib.